engine/src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -36,6 +36,9 @@ object SgeBackend {
     def script = jobDescriptor.callRootPath.resolve("script.sh")
     def returnCode = jobDescriptor.callRootPath.resolve("rc")
   }
+
+  private lazy val oldGoogleConf = GoogleConfigurationAdapter.gcloudConf.get
+  lazy val gcsConf = Try(GoogleConfiguration(oldGoogleConf.appName, oldGoogleConf.userAuthMode.getOrElse(oldGoogleConf.cromwellAuthMode)))
```

Comment:
That is sort of how it will work per the proposals [here](https://github.com/broadinstitute/cromwell/pull/627#issuecomment-203502798).